app/models/post_analyzer.rb

```diff
@@ -126,7 +129,7 @@ def raw_links
 
   # How many links are present in the post
   def link_count
-    raw_links.size
+    raw_links.size + @onebox_urls.size
```

Comment:
Why that change?

Reply:
Some links were either in the `raw_links` array or in the `@onebox_urls` array, so `link_count` was returning only a partial count of the links present in the post.